test/unit/flash_attention/flash_attention_decode/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024 - 2025 Codeplay Software Ltd. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+# list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its
+# contributors may be used to endorse or promote products derived from
+# this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# TODO(Codeplay): Re-enable Flash Decode tests once IGC issue is fixed in the CI.
+if(SYCL_INTEL_TARGET AND NOT CUTLASS_SYCL_RUNNING_CI)
+
+cutlass_test_unit_add_executable(
+  cutlass_test_unit_flash_attention_decode_xe
+  xe_flash_decode_bf16.cpp
+  xe_flash_decode_fp16.cpp
+)
+
+add_custom_target(
+  cutlass_test_unit_flash_attention_decode
+  DEPENDS
+  cutlass_test_unit_flash_attention_decode_xe
+)
+
+add_custom_target(
+  test_unit_flash_attention_decode
+  DEPENDS
+  test_unit_flash_attention_decode_xe
```

Comment:
Where is `test_unit_flash_attention_decode_xe` defined?

Reply:
Calling the `cutlass_test_unit_add_executable` for `cutlass_unit_flash_attention_decode_xe` creates `test_unit_flash_attention_decode_xe` by calling `cutlass_add_executable_tests` [here](https://github.com/codeplaysoftware/cutlass-sycl/blob/a0e4b38694a938d0c53aad45f8722867359dd490/test/unit/CMakeLists.txt#L132C3-L132C31) which is defined in the main [CMakeLists.txt](https://github.com/codeplaysoftware/cutlass-sycl/blob/a0e4b38694a938d0c53aad45f8722867359dd490/CMakeLists.txt#L882)